contrib/qemu/template.py

```diff
@@ -36,7 +36,7 @@
     "--disable-bsd-user",
     "--disable-werror",
     "--disable-xen",
-    "--audio-drv-list=pa,jack,sdl",
+    "--audio-drv-list=pa,pipewire,jack,sdl",
```

Comment:
hm, maybe we should drop pa from the list now that pipewire is in

Reply:
i can't get it to work lol `pipewire: Failed to initialize PW contextaudio: Could not init `pipewire' audio driver`

that said, the reason i didn't drop pa is because it's not automatic; you manually do -audio/-audiodev with an explicit driver, so removing pa means everyone has to update what they pick (in contrast to a random application that just automatically uses whatever you compiled it with and just connects to audio server)

Reply:
ok, fair